Weather.js

```diff
@@ -0,0 +1,28 @@
+
+  import React, { Component } from "react";
+
+
+
+
+export default class Weather extends Component {
+
+    getWeather = async (city) => {
+        const url = `https://api.weatherbit.io/v2.0/subscription/usage?key=${process.env.REACT_APP_WEATHER_KEY}&q=${city}&format=json`;
+        // try {
+        //   let response = await axios.get(url)
+        //   this.setState({ location: response.data[0]}, this.getMapURL)
+        // } catch (e) {
+        //   console.error(e);
+        //   this.setState({ error: true })
+        // }
+      }
```

Comment:
Hi there Kirk!

Feedback here is to work through task cards in order, step by step, then move on to frontend. 
You have the right idea so far:

- Define.specify the port you are listening on first (backend localhost address etc.) 
- Then you can use this variable since you need to tell your frontend which route to hit on the backend. in this case it might look something like:
- ` ${server_PORT}/weather?searchQuery=${exploreQuery}&lat=${lat}&lon=${lon}`;` (not this exactly of course as it depends on how you have specified lat and lon)
- Then use axios to do a GET request, it says, hey server! Give me something :)
- Then you can set the state to response data